pkg/restapi/governance/operation/operations.go

```diff
@@ -233,13 +231,14 @@ func (o *Operation) issueCredentialHandler(rw http.ResponseWriter, req *http.Req
 	}
 
 	// set credential status
-	credential.Status, err = o.vcStatusManager.CreateStatusID()
-	if err != nil {
-		commhttp.WriteErrorResponse(rw, http.StatusInternalServerError, fmt.Sprintf("failed to add credential status:"+
-			" %s", err.Error()))
-
-		return
-	}
+	// TODO add status ID
```

Comment:
@fqutishat what's this?

Reply:
this for governance VC I need to add status ID

Reply:
Next PR